pkg/kubelet/kuberuntime/labels.go

```diff
@@ -88,8 +88,14 @@ func newPodLabels(pod *v1.Pod) map[string]string {
 }
 
 // newPodAnnotations creates pod annotations from v1.Pod.
+// Deep copy from pod.Annotations to avoid concurrent map read and map write
```

Comment:
Nit: Like other similar logic, it seems that the comments here can be removed.

Reply:
Good idea, the comments have been removed.